### PR TITLE
[WFLY-15800] Use CLI script rather than ConfigMap

### PR DESCRIPTION
**Pull request by kabir**
to set up properties pointing to the service binding ones

https://issues.redhat.com/browse/WFLY-15800

### CODE REVIEW

File: shared-doc/attributes.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // Enable the following flag to build README.html files for EAP XP product builds.
 // Comment it out for WildFly or JBoss EAP product builds.
 //***********************************************************************************
-//:EAPXPRelease:
+:EAPXPRelease:
```
Comment (emmartins) [inline shared-doc/attributes.adoc:17]: revert please

**Review by emmartins — CHANGES_REQUESTED** [review]

**Review by emmartins — APPROVED** [review]

File: microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc [hunk 2]
```diff
@@ -60,12 +47,42 @@ Make a copy of the client ID to store in a safe place. Credentials won't appear
...
+
+These properties from the MicroProfile Config subsystem have the same names as some of the ones we already defined in link:src/main/resources/META-INF/microprofile-config.properties[microprofile-config.properties]. These get merged into one overall configuration. Where the same value exists in both places the one added in the CLI script above takes precedence, so `mp.messaging.connector.smallrye-kafka.bootstrap.servers` from the values provided by CLI script will be used. This is because the CLI script uses a higher `ordinal` when adding these properties.
+Once deployed this will store the ConfigMap settings in the application pod's `/etc/config/reactive-messaging-properties` directory.
+
+The CLI script script will get run as part of initialising the application pod, when we we pass in the environment variable `QS_USE_RHOSAK=true` when building the application as we will see later. For the bootable jar with RHOSAK combination there is a `src/main/scripts/rhoasak/bootable-jar/initialise-server.cli` script that does the same.
```
Comment (jstourac) [inline microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc:88]: double `script script` typo

Comment (jfdenise) [inline microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc:88]: "script script" ==> script
"we we" ==> we

File: microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc [hunk 3]
```diff
@@ -60,12 +47,42 @@ Make a copy of the client ID to store in a safe place. Credentials won't appear
...
 Next, we need to grant access to our service account. This is done by running the following command
 ----
 rhoas kafka acl grant-access --consumer --producer     --service-account <client id> --topic "*" --group "*"
 ----
-Substitute <client id> with the `Client ID` reported by the earlier `rhoas cluster connect` command.
\ No newline at end of file
+Substitute <client id> with the `Client ID` reported by the earlier `rhoas cluster connect` command.
+
+== Configuring your application
+The final part of the setup is to map the created service bindng config map to our application. As mentioned, once we run `rhoas cluster bind`, we will get a Config Map with entries mapped to files under the /bindings/kafka-config` directory in the application pods.
```
Comment (jfdenise) [inline microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc:59]: "bindng" ==> binding

File: microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc [hunk 4]
```diff
@@ -60,12 +47,42 @@ Make a copy of the client ID to store in a safe place. Credentials won't appear
...
-
-The `src/main/scripts/rhosak/s2i/initialize-server.cli` script will get run as part of initialising the application pod, and adds the mentioned `/bindings/kafka-config` and `/etc/config/reactive-messaging-properties` folders as config sources in the `microprofile-config-smallrye` subsystem, when we we pass in the environment variable `QS_USE_RHOSAK=true` when building the application as we will see later. For the bootable jar with RHOSAK combination there is a `src/main/scripts/rhoasak/bootable-jar/initialise-server.cli` script that does the same.
+The `rhoas cluster connect` output will also tell you to run `rhoas cluster bind`. However, we need to wait until we have deployed our application with `helm install` in the next step. Once we run `rhoas cluster bind --binding-name kafka` after  deploying our application, the Service Binding Operator will create a config map bound to the `/bindings/kafka-config` directory in the application pods. We will use this service binding in the next section, although we need to wait before we actually perform this binding.
```
Comment (jfdenise) [inline microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc:50]: "--binding-name kafka" ==> "--binding-name kafka-config"

**Review by jfdenise — CHANGES_REQUESTED** [review]

**Pull request merged**